ansible_ai_connect/ai/api/views.py

```diff
@@ -1167,6 +1171,33 @@ def __init__(self):
         else:
             logger.debug("Chatbot is not enabled.")
 
+        agent_config = AgentConfig(
+            # model="anthropic/claude-3-5-haiku-latest",
+            model="llama3.2:3b-instruct-fp16",
+            instructions="You are a helpful Ansible Automation Platform assistant.",
+            sampling_params={
+                "strategy": {"type": "top_p", "temperature": 1.0, "top_p": 0.9},
+            },
+            toolgroups=(
+                [
+                    # "mcp::weather",
+                    # "mcp::github",
+                    # "mcp::fs",
+                    # "mcp::aap_api",
+                    # "mcp::gateway_api",
+                    # "builtin::websearch",
+                ]
+            ),
+            tool_choice="auto",
+            input_shields=[],  # available_shields if available_shields else [],
+            output_shields=[],  # available_shields if available_shields else [],
+            enable_session_persistence=False,
+        )
+        self.client = AsyncLlamaStackClient(
```

Comment:
JFYI, if/when this becomes a _real_ PR the instantiation and configuration of `AsyncLlamaStack` should be moved to a `ModelPipeline` implementation.